app/views/spree/admin/product_packages/index.html.erb

```diff
@@ -6,7 +6,8 @@
   <%= javascript_include_tag 'admin/product_packages/index.js' %>
 <% end %>
 
-<%= render :partial => 'spree/admin/shared/product_sub_menu' %>
+<%# This partial was remove in solidus 1.2 but that is also when Spree.solidus_version was added %>
+<%= render :partial => 'spree/admin/shared/product_sub_menu' if !Spree.respond_to?(:solidus_version) %><%= render :partial => 'spree/admin/shared/product_tabs', :locals => { :current => 'Product Packages' } %>
```

Comment:
I think the last part of the line (render product_tabs) should not be there (it's already present below so this is a duplicates)